vite.config.ts

```diff
@@ -18,6 +18,14 @@ export default defineConfig({
 					src: 'public-assets/App_Plugins/*.js',
 					dest: 'App_Plugins',
 				},
+				{
+					src: 'public-assets/tiny-mce/**.*',
```

Comment:
I've spent time on this today, struggling to get it working nicely. The TinyMCE config requires a base path, which I've set to `/public-assets/tiny-mce`, in line with having stored the assets at that path. Changing the base path results in imports relative to the executing component, which we don't want, or it means importing all the Tiny dependencies into the editor component (which still doesn't want to work as the CSS references fail).

Interestingly, the rte-content stylesheet which has its path set in the mocked data resolves correctly (file at `/public-assets/css/rte-content`, mock data defines the path as `/css/rte-content`), but similar config in the editor component fails.

Highly likely I'm just not fully understanding how the build is running with respect to generating paths for static files. More than happy for someone better versed to have a look at improving this. Works for now, but absolutely agree would be better to not be committing all the Tiny files.

Reply:
Yeah, interesting. This part here copies files only when you build as far as I know, so if the file should exist at the "dest" path when you are running the dev-server, you need to put the file there manually, I'm afraid. Perhaps there isn't any way to copy files automatically from node_modules.

However, we might be able to just put the files in the "public" folder so we don't need an extra copy config here.

Reply:
I'll fiddle with it again tomorrow - it might be an option to extend the npm dev task to do a copy from node modules across to public, then leave the vite config to do the other copying for build... Will see what I can find out.

Reply:
> it might be an option to extend the npm dev task

Please consider not touching that as it gives us a lot of value that this particular task is fast. It's ok to commit the files, I was just wondering if there was another way.

Reply:
Alternatively, you could add a postinstall script that copies the files

Reply:
Yah, slept on it and was going for a postinstall task. Dev is beautifully fast, let's not change that. Also only needs to run once so post-install makes sense.

Reply:
@iOvergaard, I think I've come to a decent solution using postinstall:
 - copies required assets from node_modules/tinymce across to public/tinymce
 - as part of the copy, files are minified if required
 - RTE editor can now use /tinymce as the base path since Vite manages /public mapping for us
 - removes need for Vite doing any extra copying on build as assets are already in /public and copied automatically
 - no more need to commit the TinyMCE assets - they are generated as part of postinstall, then available for the build

Does mean that after this is merged team will need to run npm install, but that isn't a big deal - would be required when ever any work adds a new dependency.